src/codegen/src/main/scala/PySparkWrapperTest.scala

```diff
@@ -261,7 +261,7 @@ class SparkTransformerWrapperTest(entryPoint: Transformer,
         case "ComputePerInstanceStatistics" => computeStatisticsString(entryPointName)
         case "IndexToValue" => indexToValueString(entryPointName)
         case "ValueIndexerModel" => valueIndexerModelString(entryPointName)
-        case "_CNTKModel" | "FastVectorAssembler" | "MultiNGram" | "ImageFeaturizer"
+        case "_CNTKModel" | "FastVectorAssembler" | "MultiNGram" | "ImageFeaturizer" | "ImageAugmenter"
            | "_ImageTransformer" | "UnrollImage" | "HashTransform" | "StopWordsRemoverTransform"
```

Comment:
Agreed, this is because of the python tests, hopefully we can fix this in a separate PR with a good fuzzing test framework